contracts/factory/RegistryFactory.sol

```diff
@@ -55,19 +55,41 @@ contract RegistryFactory is Stakeable, INexusFactory {
     }
 
     function addAttester(address attester) external onlyOwner {
+        // Add the new attester to the storage array
         attesters.push(attester);
-        LibSort.sort(attesters);
+
+        // Copy the storage array into memory for sorting
+        address[] memory attestersMemory = attesters;
+
+        // Sort the memory array
+        LibSort.sort(attestersMemory);
+
+        // Copy the sorted memory array back to the storage array
+        for (uint256 i = 0; i < attestersMemory.length; i++) {
+            attesters[i] = attestersMemory[i];
+        }
     }
 
     function removeAttester(address attester) external onlyOwner {
+        // Find and remove the attester by swapping it with the last element and popping the array
         for (uint256 i = 0; i < attesters.length; i++) {
             if (attesters[i] == attester) {
                 attesters[i] = attesters[attesters.length - 1];
                 attesters.pop();
                 break;
             }
         }
-        LibSort.sort(attesters);
+
+        // Copy the storage array into memory for sorting
+        address[] memory attestersMemory = attesters;
+
+        // Sort the memory array
+        LibSort.sort(attestersMemory);
+
+        // Copy the sorted memory array back to the storage array
+        for (uint256 i = 0; i < attestersMemory.length; i++) {
+            attesters[i] = attestersMemory[i];
+        }
```

Comment:
directly assigning attesters = sorted; after sorting won’t work as expected because you cannot directly assign a memory array to a storage array this way in Solidity